classes.py

```diff
@@ -15,21 +15,27 @@ class Dice:
     current_throw : list
         List of integers of the five dice
         
```

Comment:
I just ran your files and it works super well, even the GUI ! :) This is not a suggestion, but more like if you plan on working further on it after the deadline, it would be cool to have browser GUI (which is much too much for now but just if you feel like doing it :D), because then also people with python could play it :)

Reply:
Hey Dana, same as last week this appears an operator-specific game for windows only. As I can't get it to work on my Mac. The GUI runs, it allows me to input names for multiple players, however, then the main screen fails to complete one turn. If you want me to send you the point it gets up to let me know? I don't know if it's something that'll change when its compiled after all.

Reply:
Anne, thanks for the feedback! It is really strange because it works for me.  Can you maybe send a picture of where it goes wrong? And the error message? :)